### DISCUSSION TITLE
If I throw a TextBundle in the fsnotes folder on iCloud drive, will fsnotes pick it up.

### DISCUSSION THREAD

**Discussion by apastuszak** [glushchenko/fsnotes#1581] [General]
Not gonna lie here. I'm playing the new Bear 2.0. And I'm pretty sure I'm done playing with it and want to copy my delta back to fsnotes.

I can export notes from Bear to TextBundle. Can I just drop those notes into the fsnotes folder, and launch fsnotes and have them automagically import, or do I have to choose FIle→Import?

🇺🇦

**Comment by glushchenko** [comment]
FIle→Import just "drops notes into the selected folder" so no any difference
